push-package.sh

```diff
@@ -1,3 +1,12 @@
 #!/bin/bash
+if [[ "${1}x" == "x" ]]; then
+  echo "Usage: ${0} <version>"
```

Comment:
Should we not just pull the version out the package.json?

I am totally fine leaving this manual though.

Reply:
fixed to pull it from `package.json` thanks for the idea!